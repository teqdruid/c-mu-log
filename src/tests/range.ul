foo($x) {
    10 > $x;
    1 < $x;
}

bar($i) {
    $x < 50;
    10 < $x;
}

main() {
    {OR:
	foo(0);
	foo(5);	
	foo(11);
    }
    bar(40);
}
