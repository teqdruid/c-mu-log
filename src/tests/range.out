the solutions for $x 0..11 the solutions for $y 9..19 the solutions for $z Any

 ^^^  Solution  ^^^ 

the solutions for $x Any the solutions for $y Any the solutions for $z 10..50

 ^^^  Solution  ^^^ 

No more solutions
